python/sqlflow_submitter/xgboost/train.py

```diff
@@ -172,4 +189,10 @@ def save_model(model_dir, model_params, train_params, feature_metas,
         train_params,
         feature_metas,
         feature_column_names,
-        label_meta)
+        label_meta,
+        feature_column_code)
+
+
+def save_local_file(booster, model_name):
```

Comment:
Can we add more description about the difference between save_local_file and save_model?
It may Include the export file path (local or remote) and the content.

Reply:
Thanks. I planned to use `save_local_file` to do 2 things: (1) save the booster; (2) save the PMML file. But this PR is not related to PMML, so I remove `save_local_file` here.